thirdeye/thirdeye-frontend/app/pods/components/rootcause-settings/component.js

```diff
@@ -58,7 +68,7 @@ export default Ember.Component.extend({
     get() {
       const start = this.get('anomalyRangeStart');
 
-      return start ? moment(+start).format(serverDateFormat) : moment().format(serverDateFormat);
+      return moment(start).format(serverDateFormat);
```

Comment:
the `+` here was handling the case where the ms timestamp is a string. just make sure that this doesn't break when you have query params initially

Reply:
yes, this is guaranteed to be in millis.